app/models/list_item.rb

```diff
@@ -1,3 +1,3 @@
-class ListItem < ApplicationRecord
+class ListItem < List
```

Comment:
so this shouldn't be changed. A list item belongs to a type of list.

Reply:
usually you would create a new class that would be considered a type of model. i.e there would be a `Todo` model or a `Grocery` model.

Reply:
Gahhhh, I started doing this!!! And then just... deleted them and, I don't know! I should have committed that change so you could have seen it. And when I deleted them, doing this change seemed like a 'maybe this could work,' but. Adding more models seemed like too much code to do something simple? That's my two cents and thought process.